pytest_variables.py

```diff
@@ -14,14 +27,20 @@ def pytest_addoption(parser):
         action='append',
         default=[],
         metavar='path',
-        help='path to test variables JSON file.')
+        help='path to test variables JSON/HJSON/YAML file.')
```

Comment:
We might have more formats in the future, so let's drop them from the help string.